core/src/test/scala/com/rackspace/com/papi/components/checker/ValidatorWADLHeaderSuite.scala

```diff
@@ -94,7 +94,7 @@ class ValidatorWADLHeaderSuite extends BaseValidatorSuite {
   //
   //  Like validator_Header, but with header defaults enabled
   //
-  val validator_HeaderDefaults = Validator((localWADLURI,
+  lazy val validator_HeaderDefaults = Validator((localWADLURI,
```

Comment:
These changes were due to hitting the method size limit in Java (64 KB). All of this XML in the "constructor" adds up! Changing these `val` to `lazy val` effectively puts each value into its own method.